frontend/test/model/EBEventTest.scala

```diff
@@ -6,25 +6,45 @@ import model.EventbriteDeserializer._
 import org.joda.time.Instant
 import play.api.test.PlaySpecification
 import utils.Resource
+import utils.Implicits._
 
-class EBEventTest extends PlaySpecification {
+object EBEventTest {
   val event = Resource.getJson("model/eventbrite/owned-events.2014-10-24.PROD.page-1.json")
   val ebResponse = event.as[EBResponse[EBEvent]]
+}
+
+class EBEventTest extends PlaySpecification {
+  import EBEventTest.ebResponse
   // val ebSoldOutEvent = ebResponse.data.find(_.id == "13043179501").get
   // val ebLiveEventTicketsNotOnSale = ebResponse.data.find(_.id == "11583080305").get
-  val ebLiveEvent = ebResponse.data.find(_.id == "12104040511").get
-  val ebDraftEvent = ebResponse.data.find(_.id == "13607066101").get
-  val ebCompletedEvent = ebResponse.data.find(_.id == "13125971133").get
-  val ebCancelledEvent = ebResponse.data.find(_.id == "13087550215").get
-  val nonTicketedEvent = ebResponse.data.find(_.id == "13602460325").get
-  val soldOutEvent = ebResponse.data.find(_.id == "12238163677").get
-  val startedEvent = ebResponse.data.find(_.id == "12972720757").get
-  val completedEvent = ebResponse.data.find(_.id == "13024577863").get
-  val limitedAvailabilityEvent = ebResponse.data.find(_.id == "12718560557").get
+  def ebLiveEvent = ebResponse.data.find(_.id == "12104040511").get.cheatyMigrate
+  def ebDraftEvent = ebResponse.data.find(_.id == "13607066101").get.cheatyMigrate
+  def ebCompletedEvent = ebResponse.data.find(_.id == "13125971133").get.cheatyMigrate
+  def ebCancelledEvent = ebResponse.data.find(_.id == "13087550215").get.cheatyMigrate
+  def nonTicketedEvent = ebResponse.data.find(_.id == "13602460325").get.cheatyMigrate
+  def soldOutEvent = ebResponse.data.find(_.id == "12238163677").get.cheatyMigrate
+  def startedEvent = ebResponse.data.find(_.id == "12972720757").get.cheatyMigrate
+  def completedEvent = ebResponse.data.find(_.id == "13024577863").get.cheatyMigrate
+  def limitedAvailabilityEvent = ebResponse.data.find(_.id == "12718560557").get.cheatyMigrate
+
+  def ticketedEvent = ebLiveEvent
 
-  val ticketedEvent = ebLiveEvent;
+  def updateEventWithDescription(event: EventWithDescription, desc: String) = event.copy(ebDescription = event.ebDescription.copy(description = desc))
 
-  def updateEventWithDescription(event: EBEvent, desc: String) = event.copy(description = Some(EBRichText(desc,desc)))
+  "structured event" should {
```

Comment:
this is the only test for the new event type, the old test sare all still there and designed to use the old data and fill in the new description field with what I think eventbrite does when it's migrating the old events
We need to add some new tests when we decide how to model the image url etc using the new description field.